modules/ephemeral-storage-creating-configmaps-across-namespaces.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * work_with_shared_resources/using-shared-resource-csi-driver.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="ephemeral-storage-creating-configmaps-across-namespaces_{context}"]
+= Creating a SharedConfigMap instance
+
+You can create a `SharedConfigMap` CR instance for the `ConfigMap` object that you want to share across namespaces. 
+
+.Prerequisites
+
+You must have permissions to perform the following actions:
+
+* Create `sharedconfigmaps.sharedresource.openshift.io` custom resource definition (CRD) at a cluster-scoped level.
+* Create a `ClusterRole` object for the `SharedConfigMap` CR.
+* Create role and role binding for the Shared Resource Container Storage Interface (CSI) Driver.
```

Comment:
Create a role and role binding for the Shared Resource Container Storage Interface (CSI) Driver.